tests/data_ingestion/test_fhir_ingestion.py

```
@@ -0,0 +1,304 @@
+import json
+import unittest
+import pandas as pd
+from pathlib import Path
+from datetime import datetime, timezone, date
```

Ruff (0.11.9)

`datetime.date` imported but unused

Remove unused import: `datetime.date`

(F401)